pycobertura/reporters.py

```diff
@@ -122,27 +138,21 @@ def format_total_statements(self, total_statements):
     def format_missed_lines(missed_lines):
         return [f"+{lno:d}" if is_new else f"-{lno:d}" for lno, is_new in missed_lines]
 
-    def determine_color_of_number(self, number):
+    @staticmethod
+    def determine_color_function_of_number(number):
         return red if number.startswith("+") else green
 
+    @classmethod
+    def colorize_number(cls, number):
```

Comment:
I always have to look what up "colorize number" means/does because I keep forgetting its purpose. Should this be an opportunity to make this more explicit? I obviously didn't pick a good name in the first place... 😅

Reply:
I tried to refactor that to make it more clear.
What it does it applies the color function onto number and if we print it it is printed in red if it starts with a plus and green if it does not start with a plus. I am not sure what could be more descriptive?

Reply:
ok I did it now in https://github.com/aconrad/pycobertura/pull/130/commits/d108aebca1a351847b3013cef815c5dbfe2b8213. But I am not sure you are happier now :)

Reply:
Meh, feel free to revert that commit, as I said in my last comment which I had forgotten to submit, no need to spend more brain cycles on it. :)